charts/dapr/charts/dapr_operator/values.yaml

```
@@ -23,7 +23,7 @@ webhookService:
   type: ClusterIP
   annotations: {}
 
-runAsNonRoot: true
+runAsNonRoot: false
```

```suggestion
runAsNonRoot: true
```

Suggest retaining true as it is more secure.